cegs_portal/search/templates/search/v1/partials/_non_targeting_reo.html

```diff
@@ -1,16 +1,44 @@
 {% load humanize %}
 {% load static %}
 
+<style>
+.tsv-button {
+    border: 2px solid #807e7e;
+    background-color: #fff;
+    color: #525050;
+    padding: 3px 6px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 10px;
+    margin: 4px 2px;
+    cursor: pointer;
+    transition-duration: 0.4s;
+}
+
+.tsv-button:hover {
+    background-color: #808080;
+    color: white;
+}
+</style>
+
 <div class="container">
     <a name="non-targeting-reos-{{ feature.accession_id }}"></a>
-    <div class="table-title" style="display: flex; align-items: center; justify-content: center">
-        <i class="bi bi-geo-alt-fill location-tab-icon"></i>
-        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" class="svg-icon">
-            <!--! Font Awesome Pro 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. -->
-            <path
-                d="M0 256L28.5 28c2-16 15.6-28 31.8-28H228.9c15 0 27.1 12.1 27.1 27.1c0 3.2-.6 6.5-1.7 9.5L208 160H347.3c20.2 0 36.7 16.4 36.7 36.7c0 7.4-2.2 14.6-6.4 20.7l-192.2 281c-5.9 8.6-15.6 13.7-25.9 13.7h-2.9c-15.7 0-28.5-12.8-28.5-28.5c0-2.3 .3-4.6 .9-6.9L176 288H32c-17.7 0-32-14.3-32-32z"
-            /></svg>
-        Proximal Regulatory Effect Observations
+    <div class="table-title" style="display: flex; align-items: center; justify-content: space-between; position: relative;">
+        <div style="display: flex; align-items: center; justify-content: center; flex-grow: 1;">
+            <i class="bi bi-geo-alt-fill location-tab-icon"></i>
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" class="svg-icon">
+                <!--! Font Awesome Pro 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. -->
+                <path
+                    d="M0 256L28.5 28c2-16 15.6-28 31.8-28H228.9c15 0 27.1 12.1 27.1 27.1c0 3.2-.6 6.5-1.7 9.5L208 160H347.3c20.2 0 36.7 16.4 36.7 36.7c0 7.4-2.2 14.6-6.4 20.7l-192.2 281c-5.9 8.6-15.6 13.7-25.9 13.7h-2.9c-15.7 0-28.5-12.8-28.5-28.5c0-2.3 .3-4.6 .9-6.9L176 288H32c-17.7 0-32-14.3-32-32z"
+                /></svg>
+            Proximal Regulatory Effect Observations
+        </div>
+    <div style="position: absolute; right: 0;">
+        <button class="tsv-button">
+            <a href="{% url 'search:download_tsv' feature.accession_id %}">TSV</a>
```

Comment:
I don't see any changes to urls.py. Also, "download_tsv" is way too generic.

Reply:
How about this? 
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/f7d6fdff-5cde-4174-891b-a12db1224d03)

Reply:
"The client side looks good. I'm curious why you didn't do any of the server-side stuff we talked about -- mainly modifying TemplateJsonView to support TSV as a new datatype."

Thanks for asking! I started with the client-side because I wanted to get that part right before diving into the server-side changes and get this feature at least working. Plus, messing with TemplateJsonView and all the other file pages you edited that I’ve never seen before felt like opening a can of worms with lots of unknowns. So, I thought let’s get the feature right first, and then I can tackle the server-side. Hope that makes sense!